.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```diff
@@ -165,6 +165,19 @@ steps:
     name: "validatePods"
     displayName: "Validate Pods"
 
+  - script: |
+      echo "validate pod IP assignment and check systemd-networkd restart"
```

Comment:
why log message says "systemd-networkd" restart

Reply:
The cmd `make test-validate-state` validates the state after restarting systemd-networkd restart for linux as a part of validating state.